src/Open3D/Visualization/Rendering/Filament/FilamentGeometryBuffersBuilder.cpp

```diff
@@ -155,7 +148,6 @@ std::tuple<VertexBufferHandle, IndexBufferHandle> PointCloudBuffersBuilder::Cons
     for (size_t i = 0; i < geometry_.points_.size(); ++i) {
         ColoredVertex& element = vertices[i];
         element.SetVertexPosition(geometry_.points_[i]);
-        element.tangent = float4VTangents[i];
         element.SetVertexColor(geometry_.colors_[i]);
     }
```

Comment:
Question: if I understand what's going on here correctly, the change is to remove the normals as a vertex attribute and moving it into a SurfaceOrientation instead. Maybe this is obvious to someone more familiar with Filament, but what is a SurfaceOrientation? The name doesn't suggest anything visual to me, and I can't find any documentation that describes what it does.  Is it Filament's way of rendering a point cloud?

Reply:
I removed normals (it's not a normals btw) because we are rendering point clouds using unlit material, so tangents make no sense to it.
About SurfaceOrientation, according in-code documentation: "The surface orientation helper can be used to populate Filament-style TANGENTS buffers." (`SurfaceOrientation.h`). So I just use them to generate tangent, bitangent and normal in form of quaternion, that would later consumed by material.

Reply:
Hm, anyway, we are using it not in optimal way, so it is just normals, in the end

Reply:
Btw, after I found a fix for rendering with existent vertex buffers, I returned tangents loading to pointclouds